converters/image_to_image/image_converter.py

```diff
@@ -1,42 +1,53 @@
 from PIL import Image
-from PIL.ImagePalette import random
+from PIL import ImageFilter
 import os
 
+FILTERS = ("BLUR", "CONTOUR", "DETAIL", "EDGE_ENHANCE", "EDGE_ENHANCE_MORE", "EMBOSS", 
+           "FIND_EDGES", "SHARPEN", "SMOOTH", "SMOOTH_MORE")
+
 class ImageConverter:
     def __init__(self, path, extension):
         self.path = path
         self.extension = extension
-
-    def image_resize(self, width=None, height=None):
-        img = Image.open(self.path)
-        if width is None or height is None:
-            width = img.width
-            height = img.height
-        img_resized = img.resize((width, height))
-        return img_resized
-
-    def image_rotate(self, img, angle):
-        img_rotated = img.rotate(angle, expand=True, fillcolor="black")
-        return img_rotated
-
-    def image_grayscale(self, img, apply):
-        if apply:
-            img_grayscaled = img.convert("L")
-            return img_grayscaled
-        return img
-
-    def image_convert(self, resize=None, rotate=None, grayscale=None):
-        img = self.image_resize()
-
-        if rotate is not None:
-            img = self.image_rotate(img, rotate)
+        self.img = Image.open(self.path)
+
+    def resize(self, measures):
```

Comment:
I think it's better to get the parameters with the names width and height, as they were.